cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -34,25 +34,13 @@
         transform: translate(6px);
     }
 
-    .tab-bar {
-        background: #ffffff;
-        border-radius: 10px;
-        padding-left: 10px;
-        padding-right: 10px;
-        box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
-    }
-
-    .content-container {
+    .experiment-content-container {
```

Comment:
I think this is basically the same as the global .container class, but with properties in a different order?

Reply:
It looks the same but it is different from the global container class. The container on the experiment page is specific to the interactions between the layout of the other containers on the page.